psydac/api/ast/fem.py

```diff
@@ -211,36 +278,41 @@ def __init__(self, expr, terminal_expr, spaces, tag=None, **kwargs):
             is_linear           = True
             tests               = expr.test_functions
             fields              = expr.fields
-            mapping             = spaces.symbolic_mapping
-            is_rational_mapping = spaces.is_rational_mapping
+            is_broken           = spaces.symbolic_space.is_broken
```

Comment:
What means 'broken' in this scope ?

Reply:
broken is used to denote if the space is broken